docs/contributing.rst

```diff
@@ -62,6 +65,27 @@ can be used to test built assets:
 .. _Wyrm: http://www.github.com/snide/wyrm/
 .. _Sphinx: http://www.sphinx-doc.org/en/stable/
 
+
+_dockerized-build::
+
+Dockerized build
+================
+
+If you have Docker available on your platform, you can get started building CSS and JS artifacts a bit faster and won't have to worry about any of the setup spilling over into your general environment.
+
+When building with Docker, we create an image containing the build dependencies, such as `SASS`_ , `Wyrm` and `Webpack`_ in the anticipated versions. Some of these are quite outdated and therefore ideal to isolate a container. The image is tagged as ``sphinx_rtd_theme:latest``.
```

Comment:
This is true without Docker as well, everything is installed via NPM. Just to be clear to the contributor, I think this should be removed.

```suggestion
```

Reply:
Since the Node version and Python versions expected for the build environment weren't previously made explicit, this comment felt very spot on :) I would always advocate having a Dockerfile in order to manifest the build environment. Not trying to make it mandatory, though, that's why it's just an additional development suggestion.

Reply:
This seems like a change for the contributing doc instead. The docker environment isn't going to be any different than the current build environment.